From f1cb4fc186dfa918b55987aebcb76e228c296daf Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 24 May 2018 15:05:51 +0800
Subject: [PATCH] soc: rockchip: all cpu support ARM

Let ARMv8 CPU that support AARCH32 selected on ARM.
But disabled by default.

Change-Id: I1db422c8ed3fcaf761c9ef9d6d5d68356696e4ab
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/soc/rockchip/Kconfig.cpu | 26 ++++++++++++++------------
 1 file changed, 14 insertions(+), 12 deletions(-)

diff --git a/drivers/soc/rockchip/Kconfig.cpu b/drivers/soc/rockchip/Kconfig.cpu
index ea880d2123aa..c6a830bb27f6 100644
--- a/drivers/soc/rockchip/Kconfig.cpu
+++ b/drivers/soc/rockchip/Kconfig.cpu
@@ -1,63 +1,65 @@
 # SPDX-License-Identifier: GPL-2.0
 
 menu "CPU selection"
-	depends on ARCH_ROCKCHIP && ARM
+	depends on ARCH_ROCKCHIP
 
 config CPU_RK312X
 	bool "RK3126/8"
+	depends on ARM
 	default y
 
 config CPU_RK3036
 	bool "RK3036"
+	depends on ARM
 	default y
 
 config CPU_RK30XX
 	bool "RK3066/8"
+	depends on ARM
 	default y
 
 config CPU_RK3188
 	bool "RK3188"
+	depends on ARM
 	default y
 
 config CPU_RK3288
 	bool "RK3288"
+	depends on ARM
 	default y
 
 config CPU_RK322X
 	bool "RK3228/9"
+	depends on ARM
 	default y
 
 config CPU_RV110X
 	bool "RV1107/8"
+	depends on ARM
 	default y
 
-endmenu
-
-menu "CPU selection"
-	depends on ARCH_ROCKCHIP && ARM64
-
 config CPU_PX30
 	bool "PX30"
-	default y
+	default y if ARM64
 
 config CPU_RK3308
 	bool "RK3308"
-	default y
+	default y if ARM64
 
 config CPU_RK3328
 	bool "RK3328"
-	default y
+	default y if ARM64
 
 config CPU_RK3366
 	bool "RK3366"
-	default y
+	default y if ARM64
 
 config CPU_RK3368
 	bool "RK3368"
-	default y
+	default y if ARM64
 
 config CPU_RK3399
 	bool "RK3399"
-	default y
+	default y if ARM64
 
 endmenu
-- 
2.35.3

